Makefile

```diff
@@ -121,6 +121,18 @@ lint-yml: $(TOOLING) ## Lint .yml files
 		-c .yamllint.yml \
 		.
 
+.PHONY: reproducible-build
+reproducible-build: build ## Check if the container is reproducible
+	@TAG=a make build
+	@TAG=b make build
```

Comment:
I think this is a bit meaningless because the container engine is just going to re-use the cached version of the image.

For example, I ran this target (with the last command removed) and then listed the images and got:

```shell
$ docker images

REPOSITORY                            TAG             IMAGE ID       CREATED          SIZE
ericornelissen/js-re-scan             a               68ba081069ae   23 minutes ago   205MB
ericornelissen/js-re-scan             b               68ba081069ae   23 minutes ago   205MB
ericornelissen/js-re-scan             latest          68ba081069ae   23 minutes ago   205MB
```

demonstrating that `ericornelissen/js-re-scan:a` and `ericornelissen/js-re-scan:b` are in fact the same image.

Reply:
This should be fixable by using the `--no-cache` option